docs/src/main/sphinx/connector/decimal-type-handling.fragment

```diff
@@ -1,10 +1,10 @@
 ### Decimal type handling
 
-`DECIMAL` types with unspecified precision or scale are mapped to a Trino
-`DECIMAL` with a default precision of 38 and default scale of 0. The scale can
-be changed by setting the `decimal-mapping` configuration property or the
-`decimal_mapping` session property to `allow_overflow`. The scale of the
-resulting type is controlled via the `decimal-default-scale` configuration
+`DECIMAL` types with unspecified precision or scale are ignored unless the
+`decimal-mapping` configuration property or the `decimal_mapping` session
+property is set to `allow_overflow`. Then such types are mapped to a Trino
+`DECIMAL` with a default precision of 38 and default scale of 0. The scale of
+the resulting type is controlled via the `decimal-default-scale` configuration
 property or the `decimal-rounding-mode` session property. The precision is
```

Comment:
```suggestion
`DECIMAL` with a default precision of 38 and default scale of 0. In order to change the scale of
the resulting type the `decimal-default-scale` configuration
property or the `decimal-rounding-mode` session property can be used. The precision is
```

Reply:
Two issues were there. Notice the session property name was incorrect.

Reply:
Thanks for taking a look, @hashhar. I just made the suggested change.